lib/network_utils.ml

```diff
@@ -759,7 +760,8 @@ struct
       else (debug "%s is NOT the DNS interface" interface; [])
     in
     let request = minimal @ set_gateway @ set_dns in
-    Printf.sprintf "interface \"%s\" {\n  request %s;\n}\n" interface (String.concat ", " request)
+    Printf.sprintf "interface \"%s\" {\n  send %s;\n  request %s;\n}\n"
```

Comment:
Using `{|...|}` could avoid the dance around escaping quotes.

Reply:
I've pushed a commit that does this - it requires some odd indentation to get the whitespace right around the newline though.

Reply:
n.b. the quote escaping was there before my change...but happy to keep the string syntax change or not, whichever you prefer :)

Reply:
I know. I'm fine with the way it is now. Neither one is idea.

Reply:
I've reverted this back to the old quote escaping syntax.